packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.tsx

```diff
@@ -352,7 +355,7 @@ const SecondaryEmailDefaultContent = () => (
         ),
       }}
     >
-      <p className="text-grey-400 text-xs my-2">
+      <p className="text-grey-400 text-xs mt-2">
```

Comment:
This had extra space at the bottom inconsistent with the rest of the rows. I think we probably never noticed since you have to add a secondary email to see it and it's minor.